samples/bot-configuration-app/nodejs/Readme.md

```diff
@@ -0,0 +1,133 @@
+---
+page_type: sample
+description: Demonstrating the feature of configurable card with type ahead search (static and dynamic) control on Adaptive Cards.
```

Comment:
Is this correct?

Please check all the readme file contents.

Reply:
Resolved